app/src/main/java/com/sdp/swiftwallet/presentation/main/fragments/MessageFragment.java

```diff
@@ -22,7 +22,7 @@
  * create an instance of this fragment.
  */
 public class MessageFragment extends Fragment {
-    private List<Contact> contacts = new ArrayList<>();
+    private final List<Contact> contacts = new ArrayList<>();
```

Comment:
maybe we should keep it at list and not final. i believe we will have to modify contacts in the future

Reply:
A final object can be modified, but not replaced by another one, so this should be no problem

Reply:
Normally it's ok because final is only for the list reference and not the list elements